examples/async/AsyncTreeSummarizeQueryDemo.ipynb

```diff
@@ -0,0 +1,183 @@
+   "metadata": {},
+   "source": [
+    "#### By default, `response_mode=tree_summarize` makes blocking LLM calls"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "d9ef0fef",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:root:> Building index from nodes: 5 chunks\n",
+      "INFO:root:> [query] Total LLM token usage: 19799 tokens\n",
+      "INFO:root:> [query] Total embedding token usage: 0 tokens\n"
+     ]
+    }
+   ],
+   "source": [
+    "start_time = time.perf_counter()\n",
+    "index.query(query_str, response_mode='tree_summarize')\n",
+    "elapsed_time = time.perf_counter() - start_time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9392d573",
+   "metadata": {},
+   "source": [
+    "It takes ~26s to finish query with response_mode=tree_summarize from 5 text chunks."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "474f82d1",
+   "metadata": {},
+   "source": [
+    "#### Pass in `use_async=True` to enable asynchronous LLM calls"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "78a02987",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:root:> Building index from nodes: 5 chunks\n",
+      "INFO:openai:message='OpenAI API response' path=https://api.openai.com/v1/completions processing_ms=2873 request_id=10074dbaadddd2c5eeeb684cff0f28fb response_code=200\n",
+      "INFO:openai:message='OpenAI API response' path=https://api.openai.com/v1/completions processing_ms=2877 request_id=c01cb4b32eaba25540535d5239f5d58a response_code=200\n",
+      "INFO:openai:message='OpenAI API response' path=https://api.openai.com/v1/completions processing_ms=2970 request_id=472f6dd817ee5a13cd38e1a4d125e51c response_code=200\n",
+      "INFO:openai:message='OpenAI API response' path=https://api.openai.com/v1/completions processing_ms=4149 request_id=2840b384b52a09b6ad8363deeabe7194 response_code=200\n",
+      "INFO:openai:message='OpenAI API response' path=https://api.openai.com/v1/completions processing_ms=4523 request_id=6327001f34ada72f0a3b40ea80b2920c response_code=200\n",
+      "INFO:openai:message='OpenAI API response' path=https://api.openai.com/v1/completions processing_ms=4897 request_id=5a5db6d32d23ba091aa02e0fd936aa51 response_code=200\n",
+      "INFO:root:> [query] Total LLM token usage: 19799 tokens\n",
+      "INFO:root:> [query] Total embedding token usage: 0 tokens\n"
+     ]
+    }
+   ],
+   "source": [
+    "start_time = time.perf_counter()\n",
+    "index.query(query_str, response_mode='tree_summarize', use_async=True)\n",
+    "elapsed_time = time.perf_counter() - start_time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "23469128",
+   "metadata": {},
+   "source": [
+    "It takes ~8s to finish building the GPTTreeIndex from 5 text chunks."
```

Comment:
maybe word this differently? will be confusing to the user if they see GPTTreeIndex here.

*"It takes ~8s to generate a response through hierarchical tree summarization"

Reply:
ooops, this is typo. Good catch.